includes/abbreviations.md

```diff
@@ -0,0 +1,30 @@
+<!-- Material for MkDocs shows a tooltip when you hover over any abbreviation in this list. -->
+<!-- https://squidfunk.github.io/mkdocs-material/reference/tooltips/#adding-a-glossary -->
+
+<!-- Please keep this list sorted from A-Z. -->
+
+*[AMI]: Amazon Machine Images
+*[AWS]: Amazon Web Services
+*[CD]: Continuous Deployment
+*[CI]: Continuous Integration
+*[CLI]: Command-Line Interface
+*[CVE]: Common Vulnerabilities and Exposures
+*[DCO]: Developer Certificate of Origin
+*[GCR]: Google Container Registry
+*[GPG]: GNU Privacy Guard
+*[HTML]: HyperText Markup Language
+*[HTTP]: Hypertext Transfer Protocol
+*[HTTPS]: Hypertext Transfer Protocol Secure
```

Comment:
Can we assume our readers know:

- HTML
- HTTP
- HTTPS (secure version of HTTP)

```suggestion
```